v-next/hardhat/src/internal/builtin-plugins/solidity-test/config.ts

```diff
@@ -121,6 +121,11 @@ export async function resolveSolidityTestUserConfig(
   testsPath = typeof testsPath === "object" ? testsPath.solidity : testsPath;
   testsPath ??= "test";
 
+  const solidityTest = {
+    rpcCachePath: "edr-cache",
```

Comment:
Shouldn't we be using a subdirectory under our [global hardhat cache directory](https://github.com/NomicFoundation/hardhat/blob/0ea038914eeec40eb6a7db532bed94a276ddda28/v-next/hardhat-utils/src/global-dir.ts#L28)?

Do you also happen to know why ["there are no cache cleanup implications."](https://github.com/NomicFoundation/hardhat/issues/6459#issuecomment-2715036312)? 

I'm a little confused as to who owns/is responsible for this cache dir. Is it hardhat or is it edr?

Reply:
cleaning is done by the user with `hardhat clean`. That cache is never automatically cleaned.

Reply:
Ah, sorry, I meant - shouldn't we use `hre.config.paths.cache` instead of the `edr-cache`?